EDITORS.rst

```diff
@@ -0,0 +1,35 @@
+Welcome
+=======
+
+Thank you for helping edit the NixOS "Weekly" Newsletter!
+
+Creating a new post
+===================
+
+
+1. Copy `template.rst <./template.rst>`_ to ``content/<year>/<number>-<title>.rst``
```

Comment:
I would make 2. be: Create a PR from `master` to `production` branch with the following template. We might even provide a link with that template included in url.

Reply:
I suppose it would be also good to say that you should commit directly to `master` branch since then the preview of the newsletter will be automatically created by travis here -> http://weekly.nixos.org/preview

I suppose you would already do this in #44 if you would know it.